strings/snake_case_to_camel_pascal_case.py

```diff
@@ -0,0 +1,52 @@
+def snake_to_camel_case(input: str, use_pascal: bool = False) -> str:
+    """
+    Transforms a snake_case given string to camelCase (or PascalCase if indicated)
+    (defaults to not use Pascal)
+
+    >>> snake_to_camel_case("some_random_string")
+    'someRandomString'
+
+    >>> snake_to_camel_case("some_random_string", use_pascal=True)
```

Comment:
Please add a few more tests...
```suggestion
    >>> snake_to_camel_case("SOME___RaNdOm_32_string", use_pascal=True)
    
    >>> snake_to_camel_case("_some_random_string_", use_pascal=True)

    >>> s = "some_random_string"
    >>>  snake_to_camel_case(snake_to_camel_case(s, use_pascal=True), use_pascal=False) == s
    True
```

Reply:
Sorry, I don't get the last test, it is to convert from snake to camel/pascal, that test is converting the snake to pascal and later try to convert from pascal to camel and compare to the original snake one.

Reply:
Yes.  We want to make sure that `decode(encode(s)) == s` proves that we can "roundtrip" if we have good input.

Like:
% `python3`
```
>>> "Lüsai".encode()
b'L\xc3\xbcsai'
>>> "Lüsai".encode().decode() == "Lüsai"
True
```

Reply:
But the function is not to encode/decode, is to transform from snake to other.

Your given example would be more like

```
>>> "Lüsai".encode()  # Convert a str into bytes
b'L\xc3\xbcsai'
>>> "Lüsai".encode().encode() == "Lüsai"  # str --> bytes --> bytes
True
```

I think the issue is the parameter use_pascal, which signals to instead of converting snake to camel, converts it to pascal instead, thought that could make it more useful and show an example of the equivalent to other languages ternary operator. I could remove it or create snake_to_camel_case(input) and snake_to_pascal_case(input).

If you consider it necessary I can add Camel/Pascal to snake